scripts/document.sh

```diff
@@ -60,10 +60,10 @@ jazzy \
     --documentation="docs/guides/*.md" \
     --root-url "${BASE_URL}/navigation/${RELEASE_VERSION}/" \
     --theme ${THEME} \
-    --output ${OUTPUT}
+    --output ${OUTPUT} \
+    --title "Navigation SDK for iOS" \
```

Comment:
Can we move this configuration setting to jazzy.yml?

Reply:
Yes, but then I think we won't be able to access `${RELEASE_VERSION}`...?

Reply:
Just move `title` to jazzy.yml, but keep `module_version` here.